### PR TITLE
Fix vault.inc documentation

### PR DESCRIPTION
**Pull request by HamletEagle**

### CODE REVIEW

File: plugins/include/vault.inc [hunk 1]
```diff
@@ -16,16 +16,45 @@
...
+ * @return              If len is not 0 function will return the number of characters written.
+					 If len is 0 function will return the numerical value associated with the key.	
+ */
 native get_vaultdata(const key[], data[] = "", len = 0);
 
-/* Sets a data under given key. */
+
+/**
+ * Sets data for a given key.
+
```
Comment (Arkshine) [inline plugins/include/vault.inc]: Misses a `*`

File: plugins/include/vault.inc [hunk 2]
```diff
@@ -16,16 +16,45 @@
...
+ * @param data          Data to set for the given key
+ *
+ * @noreturn
+ */
 native set_vaultdata(const key[], const data[] = "");
 
-/* Removes a key from vault.*/
+/**
+ * Removes a key and it's data from the vault.
+
```
Comment (Arkshine) [inline plugins/include/vault.inc]: Misses a `*`

File: plugins/include/vault.inc (hunk 1 above)
Comment (HamletEagle) [reply plugins/include/vault.inc]: Done

File: plugins/include/vault.inc (hunk 2 above)
Comment (HamletEagle) [reply plugins/include/vault.inc]: Done

File: plugins/include/vault.inc [hunk 3]
```diff
@@ -16,16 +16,45 @@
...
+ * Reads data from a given key.
+ *
+ *
+ * @param key           Key to get the value from
+ * @param data          Buffer to copy the value to
+ * @param len           Buffer size. If len is set to 0 then the function will 
+ * 			        	 return the value as a number
+ *
+ * @return              If len is not 0 function will return the number of characters written.
+					 If len is 0 function will return the numerical value associated with the key.
```
Comment (OciXCrom) [inline plugins/include/vault.inc]: Missing "the" before "function". Same on the line above it.

File: plugins/include/vault.inc [hunk 4]
```diff
@@ -16,16 +16,45 @@
...
+ * @param key           Key to set the value for
+ * @param data          Data to set for the given key
+ *
+ * @noreturn
+ */
 native set_vaultdata(const key[], const data[] = "");
 
-/* Removes a key from vault.*/
+/**
+ * Removes a key and it's data from the vault.
```
Comment (OciXCrom) [inline plugins/include/vault.inc]: its*

File: plugins/include/vault.inc (hunk 3 above)
Comment (HamletEagle) [reply plugins/include/vault.inc]: Fixed

File: plugins/include/vault.inc (hunk 4 above)
Comment (HamletEagle) [reply plugins/include/vault.inc]: Done

**Pull request merged**